### PR TITLE
Add  a draft version of README.md

### PR DESCRIPTION
**Pull request by linearhit**

### CODE REVIEW

File: README_.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
+# 1 BladeDISC
+
+BladeDISC is an end-to-end **DynamIc Shape Compiler** project for machine learning
+workloads, which is one of the key components of the Alibaba's [PAI-Blade](https://www.aliyun.com/activity/bigdata/blade).
+BladeDISC provides general, transparent and ease of use performance optimization
+for TensorFlow/PyTorch workloads on GPGPU and CPU backends. It supports multiple
+and flexible deployment solutions, including both Pluggin Mode inside
```
Comment (zhiqwang) [inline README_.md:7]: Hi, Is the "pluggin" here a typo, perhaps it should be plugin?

Comment (linearhit) [reply README_.md]: Thanks, it's updated.

File: README_.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+[1] The compute intensive part of the X86 backend is already supported on the
+internal version. The code decoupling is ongoing and will be open-sourced soon,
+same for the end-to-end usability.
+
+### 1.1.3 Deployment Solutions
+
+* Plugin Mode - BladeDISC works as a pluggin of TensorFlow or PyTorch. Only the
+  supported Ops are clustered and compiled, and the unsupported ones will be
+  executed by the the host runtime. We recommand this mode to most of the users
```
Comment (wyzero) [inline README_.md:52]: Do we need to give a brief definition to the `host runtime` first? `host` may refer to CPU execution for some GPU developers, leading to misunderstanding.

Or just try to use `original runtime`? example:
```
and the remaining ops will not be touched by BladeDISc and left to the original runtime.
```

File: README_.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+## 1.1 Features and Roadmap
+
+### 1.1.1 Frontend Framework Support Matrix
+
+|           | TensorFlow [1] | Pytorch [2]  |
+|---------- | -------------- | ------------ |
+| Inference |    Yes         |    Yes       |
+|  Training |    Yes [3]     |  Ongoing     |
+
+[1] TensorFlow ver1.12, 1.15, 2.4 & 2.5 are supported and fully verified. For
```
Comment (Yancey1989) [inline README_.md:27]: > `ver1.12 `

`ver` is not Tensorflow version prefix, maybe `1.12` is fine.

File: README_.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+# 2 QuickStart
+
+## 2.1 For TensorFlow Users
+
+* Step 1, Please make sure you have installed [Nvidia
+  Docker](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
+  on your host.
+
+* Step 2, It is recommended to fetch the latest BladeDISC runtime Docker
```
Comment (Yancey1989) [inline README_.md:88]: maybe we can add a command to launch the Docker container, that users can run the demo step by step.

File: README_.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+[examples](/docs/example/tensorflow/BERT_inference.md), static compiler
+optimization (XLA) shows severe performance degradation due to its compilation
+overhead, while DISC shows a 1.75x speedup.
+
+| TensorFlow  |    XLA    |    DISC    |
+|-------------|-----------|------------|
+|   1.78 s    |   41.69s  |    1.02s   |
+|   1X        |           |    1.75X   |
+
+# 2 QuickStart
```
Comment (Yancey1989) [inline README_.md:80]: Maybe it's too long in the README on the project root. I prefer a **quick view** section that introduces the simple BladeDISC API for Pytorch and TensorFlow wrapper, that users can force on **WHAT** is BladeDISC, and we can reference **HOW** to use it step by step in a separate file.

The `quick view` section can use some pseudocode just like the following:

```
# enable BladeDISC on TensorFlow program
import tensorflow_blade_disc as disc
disc.enable()

# construct TensorFlow Graph and run it
g = tf.Graph()
with g.as_default():
    ...
    with tf.session as sess:
        sess.run(...)
```

File: README_.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+By evaluating BladeDISC using a set of typical machine learning workloads for
+production purpose, DISC shows up to 3x speedup comparing with TensorFlow/PyTorch.
+
+<div align="centor">
+<img src=/docs/numbers.png width=60% />
+</div>
+
+### 1.2.1 Advantage in Dynamic Shape Workloads
+
+Specifically, for the BERT large inference on T4 we provide in the
```
Comment (Yancey1989) [inline README_.md:70]: better to reference a link to BERT large.

File: README_.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
+# 1 BladeDISC
```
Comment (Yancey1989) [inline README_.md:1]: I'm not sure do we need the **number** on the title, the suggestion layout in https://google.github.io/styleguide/docguide/style.html#document-layout is as the following:

``` text
# Document Title

Short introduction.

[TOC]

## Topic

Content.

## See also

* https://link-to-more-info
```

File: README_.md (hunk 2 above)
Comment (linearhit) [reply README_.md]: Yes indeed, done

File: README_.md (hunk 3 above)
Comment (linearhit) [reply README_.md]: done

File: README_.md (hunk 4 above)
Comment (linearhit) [reply README_.md]: I agree with the next comment, thus quickstart_tensorflow.md is recovered. Pls help to update the command for launching the docker, together with the namespace of the docker image.

File: README_.md (hunk 5 above)
Comment (linearhit) [reply README_.md]: agree, done

File: README_.md (hunk 6 above)
Comment (linearhit) [reply README_.md:72]: the link of /docs/example/tensorflow/BERT_inference.md has been provided, in which the link of the model is provided.

File: README_.md (hunk 7 above)
Comment (linearhit) [reply README_.md]: done

**Review by Yancey1989 — APPROVED** [review]
LGTM, some links are 404, but maybe we can merge this PR and keep on updating in the future.

**Pull request merged**